runtimes/spiritnet/src/weights/pallet_balances.rs

```diff
@@ -1,25 +1,10 @@
-// KILT Blockchain – https://botlabs.org
```

Comment:
license is missing

Reply:
added: [faf9f53](https://github.com/KILTprotocol/kilt-node/pull/527/commits/faf9f53336ac0e20ded5450c3ef2b1b696874cb4)